### PR TITLE
AI bot content page #2

### LINKED ISSUE #2: E7 - As a user i want to read more about the Super-bots mentioned on the website.
## To-Do
- Create content on what the 4 bots are.
- Create page and include more details about the bots
- Link the more button of each bot to the relevant section of the newly created page

## Expected output
The user should be able to read more about the bots my clicking the 'more' link relevant to each bot
 
## Impact
Hope page
Creation of new page

### PR DESCRIPTION
**Pull request by Anuranie**
## Changes proposed in the pull request
  * Add page with AI bot description
  * Add redirection to AI bot page

### CODE REVIEW

File: src/aiBots.html [hunk 1]
```diff
@@ -0,0 +1,667 @@
...
+            </div>
+        </div>
+        <!-- End Product description ver3 -->
+
+        <div class="container-fluid">
+            <div class="row pt30 bg-border-color">
+
+                <div class="container">
+
+
```
Comment (psnmissaka) [inline src/aiBots.html]: extra white space

File: src/aiBots.html [hunk 2]
```diff
@@ -0,0 +1,667 @@
...
+                                        <p>Maturify understands that each individual is unique and has their own pace
+                                            and means of learning and developing themselves. This is why we include this
+                                            particular bot which creates customized continuous development plans for all
+                                            employees, based on the knowledge they are assigned to learn.</p>
+                                        <p>The bot analyses each individual’s learning pace and capacity and develops a
+                                            knowledge growth plan specifically designed for them. Once the plan is
+                                            created is nudges these employees to stick to the timeline, thus ensuring
+                                            steady growth in the knowledge perspective for the individual and constant
+                                            risk mitigation from the company’s point of view.</p>
+
```
Comment (psnmissaka) [inline src/aiBots.html]: we can remove this white space as well right?

File: pages/automagical.html [hunk 3]
```diff
@@ -0,0 +1,624 @@
...
+                            <div class="product-description-thumb">
+                                <img src="img/product-solution.png" alt="challenge" class="shadow-image">
+                            </div>
+                        </div>
+
+                    </div>
+
+                    <div class="product-description-border"></div>
+
+
```
Comment (psnmissaka) [inline pages/automagical.html]: could remove this whitespace

File: pages/automagical.html [hunk 4]
```diff
@@ -0,0 +1,624 @@
...
+                        <div class="col-lg-5 col-md-5 col-sm-12 col-xs-12">
+                            <div class="product-description-content">
+                                <div class="heading">
+                                    <h4 class="h1 heading-title">Automagical Gaming Intelligence</h4>
+                                    <div class="heading-line">
+                                        <span class="short-line"></span>
+                                        <span class="long-line"></span>
+                                    </div>
+                                    <p>Consectetuer adipiscing elit, sed diam nonummy nibham liber tempor cum
```
Comment (psnmissaka) [inline pages/automagical.html]: what's this dummy text?

File: src/aiBots.html [hunk 5]
```diff
@@ -0,0 +1,667 @@
...
+                            </div>
+
+                        </div>
+
+                    </div>
+
+                </div>
+
+
```
Comment (psnmissaka) [inline src/aiBots.html]: extra white space

File: pages/automagical.html (hunk 4 above)
Comment (Anuranie) [reply pages/automagical.html]: This page was there originally and could be used as a template later on

**Pull request merged**